src/machine/machine_avr.go

```diff
@@ -41,9 +41,20 @@ func (p GPIO) Set(value bool) {
 		}
 	} else { // clear bits
 		if p.Pin < 8 {
-			*avr.PORTB &^= 1 << p.Pin
+			*avr.PORTD &^= 1 << p.Pin
 		} else {
 			*avr.PORTB &^= 1 << (p.Pin - 8)
 		}
 	}
 }
+
+// Get returns the current value of a GPIO pin.
+func (p GPIO) Get() (value bool) {
```

Comment:
The named return value is unnecessary in this case, as you're always returning directly. So it can simply be `func (p GPIO) Get() bool {`.

Reply:
Also done earlier.